ARExperiment/ARExperiment/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  ARExperiment
+//
+//  Created by Berta Devant on 02/01/2018.
+//  Copyright © 2018 Berta Devant. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
```

Comment:
![giphy-1](https://user-images.githubusercontent.com/797790/35443518-af705188-02ab-11e8-83c3-06d7f7f13eb3.gif)
delete all the code!

Reply:
👍